firmware/controllers/settings.cpp

```diff
@@ -884,14 +877,9 @@ static void getValue(const char *paramStr) {
 		efiPrintf("invertCamVVTSignal=%s", boolToString(engineConfiguration->invertCamVVTSignal));
 	} else if (strEqualCaseInsensitive(paramStr, "isHip9011Enabled")) {
 		efiPrintf("isHip9011Enabled=%d", engineConfiguration->isHip9011Enabled);
-	}
-
-#if EFI_RTC
```

Comment:
It seemed more appropriate to obey the command but print about lack of RTC capability, than to report "Invalid ..." -- perhaps it's better to change "Invalid" to "Unsupported" and not keep this change, versus keeping/spreading this paradigm to the other conditionally compiled things?